runtime-config-linux.md

```diff
@@ -201,6 +201,8 @@ The following parameters can be specified to setup the controller:
 
 * **`kernel`** *(uint64, optional)* - sets hard limit for kernel memory
 
+* **`kernelTCP`** *(uint64, optional)* - sets hard limit for kernel memory in tcp using
```

Comment:
using… what?  We still have this unfinished sentence [in the current tip][1].

[1]: https://github.com/opencontainers/runtime-spec/blame/c337cd2c0a425ac9a7e17a624ac3be23a02fe53f/config-linux.md#L262